actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseEnrollmentActor.java

```diff
@@ -160,6 +176,21 @@ private void unenrollCourseBatch(Request actorMessage) {
     sender().tell(result, self());
     UserCoursesService.sync(updateAttributes, userCourseResult.getId());
     generateAndProcessTelemetryEvent(request, "user.batch.course.unenroll");
+
```

Comment:
move complete code base to separate method.

Reply:
done